library/general/gatewayclassnamespaces/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: gatewayclassnamespaces
+spec:
+  crd:
+    spec:
+      names:
+        kind: GatewayClassNamespaces
```

Comment:
Could we add the schema for the parameters here?

Reply:
Updated, thanks for catching that!